### DISCUSSION TITLE
Usual navigation keys don't work in history menu

### DISCUSSION THREAD

**Discussion by saul** [marlonrichert/zsh-autocomplete#409] [General]
Hi

When in the history prompt, when I try to use `Home`/`End`, they do nothing. Similarly using `Ctrl-Left`/`Ctrl-Right` does not move backward/forward a word - they act as `PgUp` and `PgDown` and navigate the history menu instead.

I noticed in the README that:

> most other keys | Accept selection, then perform usual action

Do I need to manually setup these keybindings? What am I doing wrong?

**Comment by marlonrichert** [comment]
This is a bug and should be fixed: #411
